cmake/config/template-arguments.in

```diff
@@ -11,12 +23,12 @@ MPI_SCALARS     := { int;
                      double;
                      long double }
 
-DERIVATIVE_TENSORS := { double;
-                        Tensor<1,deal_II_dimension>;
-                        Tensor<2,deal_II_dimension> }
-
+// template names for serial vectors that we can instantiate as T<S> where
+// S=REAL_SCALARS for example
 DEAL_II_VEC_TEMPLATES := { Vector; BlockVector }
 
+// Serial vector types
+// TODO: why are parallel vectors in here?
```

Comment:
yes, historical cruft. the names no longer reflect what we do with these vectors. we ought to clean this up at some point